xml/System.Net.NetworkInformation/NetworkInterface.xml

```diff
@@ -561,28 +561,16 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The interface types are described in detail in the <xref:System.Net.NetworkInformation.NetworkInterfaceType> enumeration documentation.  
-  
- This property only returns a subset of the possible values defined in the <xref:System.Net.NetworkInformation.NetworkInterfaceType> enumeration. The possible values include the following:  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Ethernet>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Fddi>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Loopback>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Ppp>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Slip>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.TokenRing>  
-  
--   <xref:System.Net.NetworkInformation.NetworkInterfaceType.Unknown>  
-  
-   
-  
+ The interface types are described in detail in the <xref:System.Net.NetworkInformation.NetworkInterfaceType?displayProperty=nameWithType> enumeration documentation.  
+
+ In principle, this property can return any member of the <xref:System.Net.NetworkInformation.NetworkInterfaceType> enumeration. The specific value that it does return is dependent on the runtime conditions of the network topology. This means that:
```

Comment:
would change "dependent on the runtime conditions of the network topology." to "dependent on the runtime conditions of the underlying network topology."

Reply:
I agree w/ adding "underlying".